### PR TITLE
Add metadata for fdroid

### PR DESCRIPTION
**Pull request by mueller-ma**
Used to display screenshots translatable description on fdroid.

### CODE REVIEW

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This is a native Android user interface application for openHAB - an open source home automation system. The open Home Automation Bus (openHAB) project aims at providing a universal integration platform for all things around home automation. It is designed to be absolutely vendor-neutral as well as hardware/protocol-agnostic. openHAB brings together different bus systems, hardware devices and interface protocols. Supported technologies include KNX, Modbus, DMX, Philips Hue, PLCBus, Sonos, Plugwise, IHC/Elko, RFXCOM, Homematic, Koubachi, Pulseaudio, Samsung TV, Bluetooth, One-Wire, Novelan, Asterisk, generic serial devices and many others. It supports all openHAB features plus NFC tags based sitemap navigation and item control, automatic discovery of your openHAB and remote connection back to your house over the Internet with my.openHAB service. openHAB is the integration and control point of your Smart Home. You will find openHAB at http://www.openhab.org/
+
+Please note: GCM and crash analytics has been removed from the F-Droid version. Unlike in other apps it is built directly from upstream.
```
Comment (mueller-ma) [inline fastlane/metadata/android/en-US/full_description.txt]: Not sure if the last sentence is good. Often fdroid versions are forks of the upstream, so updates might receive later.

**Comment by mueller-ma** [comment]
Triggering travis

**Comment by digitaldan** [comment]
lgtm

**Comment by mueller-ma** [comment]
Ops, this broke the screenshots in the readme

**Pull request merged**